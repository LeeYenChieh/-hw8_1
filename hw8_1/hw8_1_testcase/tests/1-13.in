pc0zrLhyufT61v5jy zuTgZdKyJD5dmW6RLpgiwf98KeC2CdMF7VeWibp5l1pPYIT
pc0zrLhyufT61v5jy ybdjLVk6hliA3m3FPasauaMK8UquRhdeU9084r
pc0zrLhyufT61v5jy display
pc0zrLhyufT61v5jy dp0eGPl9E4vV3f0bAur4ScVKWatc
pc0zrLhyufT61v5jy add scrollBar scrollBar thickBlackBorder
pc0zrLhyufT61v5jy display
pc0zrLhyufT61v5jy add scrollBar thickBlackBorder thickBlackBorder
pc0zrLhyufT61v5jy zu38FdETyosq9FKob6RPXo
pc0zrLhyufT61v5jy display
pc0zrLhyufT61v5jy add scrollBar
pc0zrLhyufT61v5jy display
pc0zrLhyufT61v5jy display
pc0zrLhyufT61v5jy display
