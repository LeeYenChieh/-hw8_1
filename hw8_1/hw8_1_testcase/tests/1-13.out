ybdjLVk6hliA3m3FPasauaMK8UquRhdeU9084r 
dp0eGPl9E4vV3f0bAur4ScVKWatc scrollBar scrollBar thickBlackBorder 
zu38FdETyosq9FKob6RPXo scrollBar scrollBar thickBlackBorder scrollBar thickBlackBorder thickBlackBorder 
zu38FdETyosq9FKob6RPXo scrollBar scrollBar thickBlackBorder scrollBar thickBlackBorder thickBlackBorder scrollBar 
zu38FdETyosq9FKob6RPXo scrollBar scrollBar thickBlackBorder scrollBar thickBlackBorder thickBlackBorder scrollBar 
zu38FdETyosq9FKob6RPXo scrollBar scrollBar thickBlackBorder scrollBar thickBlackBorder thickBlackBorder scrollBar 
